modules/developers-guide/examples/graphql/skiing/1createDeployFile.graphql

```diff
@@ -0,0 +1,9 @@
+curl http://localhost:8080/graphql-admin \
```

Comment:
Shouldn't this be a `.sh` file?

Reply:
This is data federation stuff. I'll pull it out for this merge.